lib/classifier-reborn/bayes.rb

```diff
@@ -6,23 +6,26 @@
 
 module ClassifierReborn
   class Bayes
+    class CategoryNotFound < StandardError; end
```

Comment:
Excellent! Please suffix this with `Error` or `Exception`. What do you think about `CategoryNotFoundError = Class.new(StandardError)` to create them?